.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: scala
-jdk: oraclejdk7
-sudo: false
```

Comment:
Also, I'm not sure if it was god idea to get rid of these two. Let me get them back and see if it doesn't break anything.